vkubelet/apiserver.go

```diff
@@ -178,3 +180,16 @@ func (s *KubeletServer) ApiServerHandlerExec(w http.ResponseWriter, req *http.Re
 
 	remotecommand.ServeExec(w, req, s.p, fmt.Sprintf("%s-%s", namespace, pod), "", container, command, streamOpts, idleTimeout, streamCreationTimeout, supportedStreamProtocols)
 }
+
+func (s *KubeletServer) ApiServerHandlerPortForward(w http.ResponseWriter, req *http.Request) {
+	pf, ok := s.p.(PortForwarder)
+	if ok {
+		vars := mux.Vars(req)
+		namespace := vars["namespace"]
+		pod := vars["pod"]
+		portforward.ServePortForward(w, req, pf, fmt.Sprintf("%s-%s", namespace, pod), "", nil, time.Second*60, time.Second*60, []string{})
```

Comment:
suggest to call p.GetPodFullName instead of fmt.Sprintf("%s-%s", namespace, pod)